SU-SOE/stanford_soe_helper
# SOE-3304 address h spotlight responsive bug and cleaned up grunt task

# READY FOR REVIEW

# Summary
- Addressed responsive bigs for horizontal spotlights
- Cleaned up Grunts tasks (and package.json file) to avoid the need of Ruby (reduce command line errors when running Grunt)

# Needed By (Date)
- End of sprint

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. On this page, `get-involved/alumni`, ensure that the horizontal postcard block is repsonsive (addressed the issues beginning at `767px` and that padding issue is cleaned up

# Affected Projects or Products
- SoE
- stanford_soe_helper

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-3304

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
